src/style.css

```
@@ -1,3 +1,175 @@
+html {
+  --acdc-color: #00acdc;
```

create variables with root selector
```suggestion
  --main-color: #00acdc;
```